### PR TITLE
Feat merge heudiconv and bids_postprocess

### PR DESCRIPTION
**Pull request by fordmcdonald**
# Update xnat2bids Interface

Two flags have been added to the list of parameters.  These flags, if specified, dictate which component of the xnat2bids pipeline will be run.  

`--skip-export` : bypasses the DICOM export step of the pipeline, immediately launching BIDS conversion module instead
`--export-only` : solely launches DICOM export 

# Coalesce BIDS Conversion Steps Into Single Function

All the necessary steps for BIDS conversion (`run_heudiconv` and `bids_postprocess`) have been umbrellaed beneath one common function: `dcm2bids`.

### CODE REVIEW

**Comment by elizabethlorenc** [comment]
sorry if I'm missing something, but how does xnat2bids get the project and subject if you skip the export?

**Comment by fordmcdonald** [comment]
@elizabethlorenc You're right.  Thanks for catching this! Originally, I had planned to pass project and subject into xnat2bids, but I think that just complicates things for the user.  Instead, if project and subject are undefined, I'm going to fetch them with the provided session id.  I'm going to set this PR back to Draft and push the fix along with some tests.

File: tests/integration/test_export_and_heudiconv.py [hunk 1]
```diff
@@ -182,9 +182,7 @@ def test_unauthorized_user_exception_handling():
     user = "bad_user"
     password = "bad_password"
 
-    connection = requests.Session()
-    connection.verify = True
-    connection.auth = (user, password)
+    connection = establish_connection(user, password)
```
Comment (mirestrepo) [inline tests/integration/test_export_and_heudiconv.py:185]: Nice!

File: xnat_tools/dcm2bids.py
Comment (mirestrepo) [inline xnat_tools/dcm2bids.py:1]: Should we add this as a new console script in the PyProject.toml file?

**Review by mirestrepo — APPROVED** [review]
Looks great. As just discussed, lets get rid of console scripts other than `xnat2bids` and have only one interface

**Pull request merged**